### DISCUSSION TITLE
Subcircuit files

### DISCUSSION THREAD

**Discussion by prlaba** [hneemann/Digital#1111] [Q&A]
Does Digital require each subcircuit to reside in its own file?  Or can I combine multiple subcircuits in the same file, like a Logisim library?

I often create circuits with a main circuit and multiple subcircuits.  I'd like to maintain them in a single file if possible.

**Comment by hneemann** [comment]
No. Each subcircuit must be saved in a separate file. 
This is intended to simplify version control e.g. with git.
